tv_apps/browser/js/settings.js

```diff
@@ -546,7 +620,9 @@ var Settings = {
    * Handle Key Event.
    */
   handleKeyEvent: function settings_handleKeyEvent(ev) {
-    if( !Settings.isDisplayed() ) return false;
+    if( !Settings.isDisplayed() ) {
+      return;
```

Comment:
I saw other modules with handleKeyEvent still keeps returning `false`, how about just following that?